frontend/components/commons/GlobalActions.vue

```diff
@@ -76,6 +81,10 @@ export default {
       type: Object,
       required: true,
     },
+    annotationEnabled: {
```

Comment:
passing dataset to component should be enough for calculate this param. I will create an tech-debt issue for refactoring components and dataset getter fields.